src/Compiler/Utilities/Caches.fs

```diff
@@ -0,0 +1,394 @@
+// LinkedList uses nulls, so we need to disable the nullability warnings for this file.
+namespace FSharp.Compiler
+
+open System
+open System.Collections.Generic
+open System.Collections.Concurrent
+open System.Threading
+open System.Diagnostics
+open System.Diagnostics.Metrics
+
+[<Struct; RequireQualifiedAccess; NoComparison>]
+type EvictionMethod =
```

Comment:
I removed `Blocking` eviction method for now. It's another thing that would rot, because it's unused.

Reply:
Agree